README.md

```diff
@@ -1,82 +1,92 @@
-# ArcGIS Runtime SDK for .NET samples
+[![Link: ArcGIS Developers home](https://img.shields.io/badge/ArcGIS%20Developers%20Home-633b9b?style=flat-square)](https://developers.arcgis.com)
+[![Link: Documentation](https://img.shields.io/badge/Documentation-633b9b?style=flat-square)](https://developers.arcgis.com/net/)
+[![Link: Tutorials](https://img.shields.io/badge/Tutorials-633b9b?style=flat-square)](https://developers.arcgis.com/documentation/mapping-apis-and-services/tutorials/)
+![Badge: Samples](https://img.shields.io/badge/Samples-black?style=flat-square)
+[![Link: Demos](https://img.shields.io/badge/Demos-633b9b?style=flat-square)](https://github.com/Esri/arcgis-runtime-demos-dotnet)
+[![Link: Toolkit](https://img.shields.io/badge/Toolkit-633b9b?style=flat-square)](https://developers.arcgis.com/net/ui-components/)
+[![Link: Templates](https://img.shields.io/badge/Templates-633b9b?style=flat-square&logo=visualstudio&labelColor=gray)](https://github.com/Esri/arcgis-runtime-templates-dotnet)
+[![Link: NuGet](https://img.shields.io/badge/NuGet-633b9b?style=flat-square&logo=nuget&labelColor=gray)](https://www.nuget.org/profiles/Esri_Inc)
+[![Link: Esri Community](https://img.shields.io/badge/🙋-Get%20help%20in%20Esri%20Community-633b9b?style=flat-square)](https://community.esri.com/t5/arcgis-runtime-sdk-for-net/bd-p/arcgis-runtime-sdk-dotnet-questions)
 
-This project contains samples for the ArcGIS Runtime SDK for .NET including WPF, UWP and Xamarin platforms.
+# ArcGIS Runtime SDK for .NET - Samples
 
-## Samples tables of contents
+<img src="./samples_screenshot.png" width="500px" />
```

Comment:
It's a little weird that this html renders a link to the image in the repo. Not a big deal, but I wonder if you could define this as markdown to avoid that (?) ... just thinking out loud 🤔

Reply:
It is weird, I think it is a GitHub feature. It also happens with the sample readmes, which use markdown-style images. Wrapping it with a link would fix the issue, I'll update that to link to the Microsoft store when clicked.